### DISCUSSION TITLE
Icon for web browser

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#1744] [Q&A] [answered]
I created an iTwin based app/website that was bootstrapped from itwin-viewer templated.  We would like to use our app/company icon in the web browser tab (image attached) in place of the iTwin icon

![Favicon](https://user-images.githubusercontent.com/83007711/123832339-b0007b80-d91e-11eb-9d5b-656647f2e6a8.JPG)

I tried looking through code to find were this icon is specified but was unable to find.  Can you please guide me how I can display our own icon.

Thank you,

Ovais

**Comment by aruniverse** [comment, marked as answer]
If you used the itwin-viewer template it'll be public\favicon.ico
Just replace that with your own. You'll probably also want to update the logo.png and the index.html

**Reply by ovaisa** [reply]
Thank you Arun and Will, I was able to get this to work.  There was no favicon.ico in the public folder, I just added one and it seems to work (not sure where it was picking the iModel icon?).  Also there is no logo.png in public folder, but I put one there anyway, I hope it will be picked up wherever it is used.

Thanks for your help (as always!!!)...

**Reply by aruniverse** [reply]
Ah sorry, looks like we didn't add those in our template, I'll make sure to update it accordingly.

Your public folder should look like: https://github.com/imodeljs/create-react-app/tree/imodeljs/packages/cra-template-typescript/template/public

I would add what's missing and update it as needed.
